AssignAll/AssignAll.Test/UnitTests.cs

```diff
@@ -118,28 +118,30 @@ public async Task EmptyCode_AddsNoDiagnostics()
             await VerifyCS.VerifyAnalyzerAsync(test);
         }
 
-        [Fact]
-        public async Task EnableAndDisableComments_EnablesAndDisablesAnalyzerForTextSpans()
+        [Theory]
+        [InlineData("// AssignAll")]
+        [InlineData("//AssignAll")]
```

Comment:
Add test cases on trailing whitespace + combinations with tabs and regular whitespace.